src/cascadia/TerminalControl/SearchBoxControl.cpp

```diff
@@ -110,6 +110,20 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         }
     }
 
+    // Method Description:
+    // - Allows to set the value of the text to search
+    // Arguments:
+    // - text: string value to populate in the TextBox
+    // Return Value:
+    // - <none>
+    void SearchBoxControl::PopulateTextbox(winrt::hstring const& text)
+    {
+        if (TextBox())
+        {
+            TextBox().Text(text);

```

Comment:
I'm only gonna block on the smallest thing - can we have the text be selected as well? So all the user has to do is hit <kbd>bksp</kbd> to clear the input text?

Reply:
@zadjii-msft - The text gets selected already, since after the call to this we call `SetFocusOnTextbox`.  I preferred not to put it here to leave the interface clean.

Reply:
Well that's perfect then, thanks!